src/Platforms/MySQL8013Platform.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Platforms;
+
+/**
+ * Provides features of the MySQL since 8.0.13 database platform.
+ *
+ * Note: Should not be used with versions prior to 8.0.13.
+ */
+class MySQL8013Platform extends MySQLPlatform
+{
+    public function getColumnNameForIndexFetch(): string
+    {
+        return "COALESCE(COLUMN_NAME, CONCAT('(', REPLACE(EXPRESSION, '\\\''', ''''), ')'))";
```

Comment:
This looks unnecessarily complicated. Can't we just select the `COLUMN_NAME` and `EXPRESSION` columns as separate columns?

Reply:
I doesn't make sense, because expression could be a part of composite index for example: (column, column, expression, column). So in term of code it is a column

Reply:
I understand that, but that does not really answer my question.